tools/parse-pkgs.sh

```diff
@@ -182,9 +203,41 @@ KUBE_TAG=$(linuxkit_tag pkg/kube)
 # on the previous tags being already defined.
 EVE_TAG=$(synthetic_tag zededa/eve pkg/eve/Dockerfile.in)
 
+help() {
+  echo "Usage: $0 [<image.yml.in> [-p <pkg>]]" >&2
+  echo
+  echo "    substitutes *_TAG in the <image.yml.in> input file and outputs " >&2
+  echo "    results to stdout." >&2
+  echo
+  echo "  -p pkg   - Removes everything which is not the <pkg> from " >&2
+  echo "             the resulting output." >&2
```

Comment:
This seems to support a single package being patched? Can this be used to update more than one package?
If not, is the recommended workflow to run it multiple times? e.g.
make LIVE_PKG=pkg/wwan live
make LIVE_PKG=pkg/pillar live

If that is the case it would be good to include that in the documentation.